client/src/components/Header/Header.scss

```diff
@@ -0,0 +1,126 @@
+$bgColor: #e6e3e3ad;
```

Comment:
keep all variables in a separate file, like variables.scss, and import it wherever necessary

Reply:
Done